javascript/javascript1/week2/homework4/eventApp.js

```diff
@@ -0,0 +1,13 @@
+const weekDays = ['Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday', 'Sunday'];
+const today = new Date()
+function getEventWeekDay(days) {
+    const day = today.getDay();
+    let result;
+    if(day + days <7){
+        result = day + days -1;
```

Comment:
You got the right idea with the remainder operation; awesome! ⭐ Can you think of a to avoid the `if` statement? (Hint: what is `(day + days - 1) % 7` equals to when the sum is below 7?)

Another minor thing is the `-1`; can you think of a way to avoid it? (Hint: what's the first day of the week according to `getDay`?) 😄